core-developers/committing.rst

```diff
@@ -146,9 +169,6 @@ The inline Sphinx roles like ``:func:`` can be used help readers
 find more information. You can build HTML and verify that the
 link target is appropriate by using :ref:`make html <building-using-make>`.
 
-While Sphinx roles can be beneficial to readers, they are not required.
-Inline ````code blocks```` can be used instead.
```

Comment:
I deliberately removed this. The above sentence says "*should* be valid reST" (emphasis mine), so IMO we do not need to repeat that it is not required. Moreover, in most cases, we actually ask contributors explicitly add markup.